tests/sshserver.pl

```diff
@@ -764,8 +769,8 @@ sub sshd_supports_opt {
 
 if ($^O eq 'MSWin32' || $^O eq 'cygwin' || $^O eq 'msys') {
     # convert MinGW drive paths to Cygwin drive paths
-    $identity_config =~ s/^\/(\w)\//\/cygdrive\/$1\//;
-    $knownhosts_config =~ s/^\/(\w)\//\/cygdrive\/$1\//;
+    chomp($identity_config = `cygpath -u -a $identity_config`);
+    chomp($knownhosts_config = `cygpath -u -a $knownhosts_config`);
```

Comment:
These lines require a more flexible patch that does not require the cygpath utility.